.github/scripts/checkTranslationFilesOnPr.ts

```diff
@@ -0,0 +1,24 @@
+import { config, echo } from 'shelljs'
+
+config.fatal = true
+
+// matches all translation.json and InfoPlist.strings files except for the base ones
+const TRANSLATION_FILES = new RegExp(
+  /^locales\/(?!base\/)[^\/]+\/translation\.json$|^ios\/celo\/(?!Base\b)\w+\.lproj\/InfoPlist\.strings$/
+)
+
+echo('Checking modified files')
+const updatedFilenames = process.argv[2]
+const modifiedTranslationFiles = updatedFilenames.split(',').filter((filename) => {
+  return filename.match(TRANSLATION_FILES)
+})
+
+if (modifiedTranslationFiles.length > 0) {
+  echo(
+    `❌ Only base translation files should be modified! The following files should not be modified:`
+  )
+  echo(modifiedTranslationFiles.join('\n'))
+  process.exit(1)
+}
+
+echo('✅ All good to go 🚀')
```

Comment:
I think we could maybe even remove this script by relying on the `files` feature from `tj-actions/changed-files`

```
    - name: Get changed files
      id: changed-files
      uses: tj-actions/changed-files@v35
      with:
        files: |
          locales/*/translation.json
          !locales/base/translation.json
          ios/celo/*.lproj/InfoPlist.strings 
          !ios/celo/Base.lproj/InfoPlist.strings
```

Makes it a bit easier to read the paths too.

And checking the `any_changed` output from the script.